pkg/templates/runasnonroot/template.go

```diff
@@ -14,6 +14,8 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+const templateKey = "run-as-non-root"
```

Comment:
I think we need to keep `-user` as this will be a breaking change
```suggestion
const templateKey = "run-as-non-root-user"
```